src/Microsoft.Health.Dicom.Blob/Utilities/ExternalBlobDataStoreConfiguration.cs

```diff
@@ -18,4 +18,6 @@ internal class ExternalBlobDataStoreConfiguration
 
     // use for local testing with Azurite
     public string ContainerName { get; set; }
+
+    public string ServiceStorePath { get; set; } = String.Empty;
```

Comment:
How will this be used?

Reply:
This is the relative directory, right? Perhaps it should be like `Directory`? Or something a little less generic like `StorageDirectory`?